contracts/tests/test_raidenchannels.py

```diff
@@ -285,15 +285,15 @@ def test_close_call(web3, chain, contract, channel):
 
     # Cannot close what was not opened
     with pytest.raises(tester.TransactionFailed):
-        contract.transact({'from': A}).close(receiver, open_block_number, balance, balance_msg_sig)
+        contract.transact({'from': A}).uncooperativeClose(receiver, open_block_number, balance, balance_msg_sig)
     with pytest.raises(tester.TransactionFailed):
-        contract.transact({'from': sender}).close(A, open_block_number, balance, balance_msg_sig)
+        contract.transact({'from': sender}).uncooperativeClose(A, open_block_number, balance, balance_msg_sig)
 
     # Cannot close if arguments not correct
     with pytest.raises(ValueError):
-        contract.transact({'from': sender}).close(receiver, open_block_number, balance)
+        contract.transact({'from': sender}).initChallengePeriod(receiver, open_block_number, balance)
```

Comment:
since this is different from the code before, please double check, should it be initChallenge first and settleChannel then, is this correct for the test logic (might be, just not sure).

Reply:
Tests will be re-written and re-checked, hopefully better documented soon. The previous test with `close` was an attempt to make sure the number of arguments was correct. That test doesn't make sense anymore anyway because we have overloading. Current test just makes sure `initChallengePeriod` is private.
Will leave this like it is for now and I'll have a test dedicated PR.